### DISCUSSION TITLE
OpenAI temperature and seed

### DISCUSSION THREAD

**Discussion by thundergore** [MaartenGr/BERTopic#1734] [General]
Im interested in testing temperature and seed when using openAI for representation: https://platform.openai.com/docs/api-reference/chat/create

I note we've got the Llama2 temperature mentioned in the llm.md but not anything for openAI. I'm wondering if it's something to set in the client, the openAI model or maybe in the prompt itself? 

Is it possible?

**Comment by MaartenGr** [comment]
Yes, you can use the `generator_kwargs` to pass any additional parameters. For more information, you can check out the docstrings of the OpenAI representation model in BERTopic.

**Comment by thundergore** [comment]
Thank you for your response!
